src/radix_connect/well_known_client/models/dapp_definitions.rs

```diff
@@ -0,0 +1,55 @@
+use crate::prelude::*;
+
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
```

Comment:
Lacks `Clone`, **always** derive `Clone` and probably we want `PartialEq, Eq` too!